draft-ietf-core-dns-over-coap.md

```diff
@@ -292,18 +292,23 @@ A DoC server MUST be able to parse requests of Content-Format "application/dns-m
 The DoC client SHOULD set the ID field of the DNS header always to 0 to enable a CoAP cache (e.g., a CoAP proxy en-route) to respond to the same DNS queries with a cache entry.
 This ensures that the CoAP Cache-Key (see {{-coap-fetch}} Section 2) does not change when multiple DNS queries for the same DNS data, carried in CoAP requests, are issued.
 
-### Examples
+### Examples {#sec:req-examples}
 
 The following example illustrates the usage of a CoAP message to
 resolve "example.org. IN AAAA" based on the URI "coaps://\[2001:db8::1\]/". The
 CoAP body is encoded in "application/dns-message" Content Format.
+For better readability, we provide the payload in a human-readable format.
+In the actual message, however, it would be encoded in the binary message format defined in {{-dns}}.
 
     FETCH coaps://[2001:db8::1]/
     Content-Format: application/dns-message
     Accept: application/dns-message
-    Payload: 00 00 01 20 00 02 00 00 00 00 00 00 07 65 78 61 [binary]
-             6d 70 6c 65 03 6f 72 67 00 00 1c 00 01 c0 0c 00 [binary]
-             01 00 01                                        [binary]
+    Payload (human-readable):
```

Comment:
I would probably keep at least one DNS message around in both binary and readable form, so readers can find out how exactly the body looks like.

Reply:
Ok, I will put one replication with the query example (which is the first in the draft).

Reply:
See [8478567](https://github.com/core-wg/draft-dns-over-coap/pull/39/commits/847856778fd3b2cfa8b0e0ad198c02b082a3fdc3)

Reply:
(btw: the reason the binary representation is shorter below is because this little exercise of converting to dig revealed an error in the dump: there were two questions in the query, one for A, one for AAAA, see https://datatracker.ietf.org/doc/rfc9619/)